backend/entrypoint.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
+
+if [ $MODE = "feeds" ]; then
+    echo  "Starting on create feeds mode"
+    node dist/tools/create-feeds.js
+elif [ $MODE = "fund" ]; then
+    echo  "Starting on fund accounts mode"
+    node dist/tools/fund-accounts.js
+else
+    echo  "Starting on relayer mode"
+    node dist/index.js
+fi
```

Comment:
It is much more idiomatic to use positional arguments instead, such that you can do:
```
docker run -it relayer-backend create-feeds
docker run -it relayer-backend fund-accounts
# for default mode (run relayer):
docker run -it relayer-backend
```